minerl/herobraine/hero/handlers/agent/observations/inventory.py

```diff
@@ -92,14 +95,13 @@ def from_hero(self, obs):
             key = util.get_unique_matching_item_list_id(self.items, type_name, stack['metadata'])
             assert stack["quantity"] >= 0
             assert stack["metadata"] in range(16)
-            if key is not None:
+            if key is not None and self.space.contains(key):
```

Comment:
@shwang Can you confirm that with item variant squashing this is the correct behavior?

Reply:
The `self.space.contains(key)` should be guaranteed by the `key is not None`. Maybe make the `self.space.contains` into assert statement (if we keep it at all)?